src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -4125,6 +4125,31 @@ public C()
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
 
+        [Fact]
+        public void TargetTypedSwitch_StringInsert()
```

Comment:
>StringInsert [](start = 38, length = 12)

What does "StringInsert" stand for?

Reply:
The test was meant to have a string interpolation with inserts. Thanks

---
In reply to: [314123410](https://github.com/dotnet/roslyn/pull/37596#discussion_r314123410) [](ancestors = 314123410)